site/faq/README.md

```diff
@@ -4,59 +4,45 @@
 
 Please visit the [roadmap document] and the [kpt milestones].
 
-### What does kpt provide that git clone doesn't?
-
-kpt enables out-of-the-box workflows that `git clone` does not. Such as: cloning
-and versioning git subdirectories, updating from upstream by performing
-structured merges on resources, programmatically editing configuration (rather
-than with an editor), etc.
```

Comment:
Is this removal intended?
This section refers to the `kpt pkg` which is not described anywhere below. And git is so popular so it seems useful to tell the differences (even if it's verbose). Or do we want to de-emphasize the package management feature from kpt?

Reply:
Yes, this removal was intended. This was the main emphasis when we first created the kpt packaging mechanism, but isn't now. For instance, we plan to support container images as storage.

I'm trying to up-level the FAQ to the whole toolchain and the wysiwyg experience.

Reply:
ah, that makes sense.